### DISCUSSION TITLE
FreshRSS integration in FocusReader and Push Notifications?

### DISCUSSION THREAD

**Discussion by DickenSerm** [FreshRSS/FreshRSS#6321] [Q&A]
Are there people here who have integrated FreshRSS into the FocusReader app and can explain to me why I don't get push notifications about new content? Strangely, this works for subscriptions created directly in FocusReader.

Greets Dicken

**Comment by ZorzStudios** [comment]
You're at least one step ahead of me. :) I installed FocusReader, entered my server (the URL where FreshRRS is installed on my website), admin ID, and a password... There were no error messages, but the feeds are empty, sync was reported as successful, but nothing is populated. I hope I'll go past this stage and will "gladly" face your issue, if it happens. Have you solved yours, anyway?

**Reply by ZorzStudios** [reply]
Ooof, I got in! The server was _not_ the FreshRSS installation directory, but further down at /p/api/greader.php

Yes, it is all in the instructions, if a casual user like me reads them a few times. :)
